code-challenges.js

```diff
@@ -49,3 +123,17 @@ var testString2 = "jabberwocky"
 var arr1 = [3, 7, 10, 5, 4, 3, 3]
 var arr2 = [7, 8, 2, 3, 1, 5, 4]
 // Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
+
+//create a function that takes int two arrays as arguments
+const noDupes = (array1, array2) =>{
+    //combine the two arrays into one.
+    //used the spread operator to concatenate the two arrays into one.
+    combArr = [...array1, ...array2]
+    //filter through combined array
+    //indexOf returns the index of the FIRST instance of that value, so any other instance of the same value will not be returned from filter.
+    let sortedArr = combArr.filter((value, index) => combArr.indexOf(value) === index)
+    //return the sorted array with no duplicates
+    return sortedArr
+}
+
+console.log(noDupes(arr1, arr2));
```

Comment:
Awesome!! Great work! I think this is another place where I would take a look at the refactor example I showed you on Challenge 2 and see if you could do that here.